packages/core/src/components/dropdown/dropdown-vars.scss

```
@@ -1,71 +1,88 @@
-:root,
-.tds-mode-light {
-  --tds-dropdown-bg: var(--tds-grey-50);
-  --tds-dropdown-border-bottom: var(--tds-grey-600);
-  --tds-dropdown-border-bottom-hover: var(--tds-grey-800);
-  --tds-dropdown-border-bottom-open: var(--tds-blue-400);
-  --tds-dropdown-border-bottom-hover: var(--tds-grey-868);
-  --tds-dropdown-value-color: var(--tds-grey-958);
-  --tds-dropdown-label-color: var(--tds-grey-958);
-  --tds-dropdown-label-inside-color: var(--tds-grey-958);
-  --tds-dropdown-helper-color: var(--tds-grey-700);
-  --tds-dropdown-placeholder-color: var(--tds-grey-500);
-  --tds-dropdown-disabled-color: var(--tds-grey-400);
-  --tds-dropdown-filter-input-color: var(--tds-grey-958);
-  --tds-dropdown-border-radius: var(--tds-spacing-element-4) var(--tds-spacing-element-4) 0 0;
-  --tds-dropdown-list-border-radius-down: 0 0 var(--tds-spacing-element-4)
-    var(--tds-spacing-element-4);
-  --tds-dropdown-list-border-radius-up: var(--tds-spacing-element-4) var(--tds-spacing-element-4) 0
-    0;
-
-  .tds-mode-variant-primary {
-    --tds-dropdown-bg: var(--tds-grey-50);
+tds-dropdow,
+.scania tds-dropdown,
+.traton tds-dropdown {
```

If rules are shared between brands, then this one should be sufficient. There was a typo in previous one. Please check.
```suggestion
tds-dropdown {
```